src/platform/EFR32/Logging.cpp

```diff
@@ -176,7 +176,7 @@ void LogV(uint8_t module, uint8_t category, const char * aFormat, va_list v)
 
         // Form the log prefix, e.g. "[DL] "
         formattedMsg[formattedMsgLen++] = '[';
-        GetModuleName(formattedMsg + formattedMsgLen, module);
+        GetModuleName(formattedMsg + formattedMsgLen, ChipLoggingModuleNameLen + 1, module);
```

Comment:
should this be `sizeof(formattedMsg) - formattedMsgLen` ?

same comment for below - I believe that if we take a size argument, it should follow the data sizes instead of using constants.

Generally the constant seems named oddly ... should probably be `chip::Logging::kMaxModuleNameLen`. Maybe for a future PR.

Reply:
Renaming all the constants :)